centinel/views.py

```diff
@@ -171,17 +171,76 @@ def get_user_specific_content(folder, filename=None, json_var=None):
     if not client.has_given_consent:
         flask.abort(418)
 
+    # all of the scheduler files are combined together here.
+    # this is run every time the experiment list or "scheduler.info"
+    # is requested.
+    if (json_var == "experiments" and 
+        (filename is None or filename == "scheduler.info")):
+        client_scheduler_filename = os.path.join(config.experiments_dir,
+                                                 username, "scheduler.info")
+        country_scheduler_filename = os.path.join(config.experiments_dir,
+                                                  client.country, "scheduler.info")
+        global_scheduler_filename = os.path.join(config.experiments_dir,
+                                                 "global", "scheduler.info")
+
+        freqs = {}
+        if os.path.exists(client_scheduler_filename):
+            with open(client_scheduler_filename, 'r') as file_p:
+                freqs.update(json.load(file_p))
+        if os.path.exists(country_scheduler_filename):
+            with open(country_scheduler_filename, 'r') as file_p:
+                freqs.update(json.load(file_p))
+        if os.path.exists(global_scheduler_filename):
+            with open(global_scheduler_filename, 'r') as file_p:
+                freqs.update(json.load(file_p))
+
+        with open(client_scheduler_filename, 'w') as file_p:
+            json.dump(freqs, file_p)
+
     files = {}
     user_dir = os.path.join(folder, username, '*')
     for path in glob.glob(user_dir):
         file_name = os.path.basename(path)
         files[file_name] = path
 
+    # include country-specific baseline content
+    country_specific_dir = os.path.join(folder, client.country)
+    if os.path.exists(country_specific_dir):
+        # if baseline experiments exist for this country (==folder exists),
+        # sync up all of the files in that dir.
+        for path in glob.glob(os.path.join(country_specific_dir, "*")):
+            file_name = os.path.basename(path)
+            # avoid sending the country-specific scheduler here
+            # the only scheduler file sent must be the unified scheduler
+            # in the client's experiments dir.
+            if file_name == "scheduler.info":
+                continue
+            files[file_name] = path
+    else:
+        print ("Country baseline folder %s "
+               "doesn't exist!" %(country_specific_dir))
+
+    # include global baseline content
+    global_dir = os.path.join(folder, "global")
+    if os.path.exists(global_dir):
```

Comment:
As in the case of the frequencies, we should reorder this so we pull global, then country, then machine. The code is fine, it just needs to be reordered.